notable.go

```diff
@@ -17,7 +18,11 @@ func targetRoom() string {
 	return "general"
 }
 
-func Record(authorID string, category string, text string, slackToken string) {
+func Record(authorID string, category string, text string, slackToken string) error {
+	if len(strings.Trim(text, " ")) == 0 {
```

Comment:
[nb] FWIW this `strings.Trim(text, " ")` might be good to do regardless at the beginning of the method. Would we want any notable announcements to go out with leading or trailing spaces?